packages/shared/test/unit/args_tests.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-env jest */
+import { args, getArgName } from '../../args';
+
+describe('args(args)', () => {
+    it('it should parse the arguments correctly', () => {
```

Comment:
The second _it_ in `it('it ..` is redundant.

Reply:
I have given it a try, please see if what I did matches what you had in mind.